packages/ckeditor5-image/tests/imageinsert/imageinsertui.js

```diff
@@ -80,6 +80,24 @@ describe( 'ImageInsertUI', () => {
 			expect( dropdown ).to.be.instanceOf( DropdownView );
 		} );
 
+		describe( 'button', () => {
+			it( 'should have the base properties set correctly', () => {
+				const button = dropdown.buttonView;
+
+				expect( button.isOn ).to.be.false;
+				expect( button.tooltip ).to.be.true;
+				expect( button.label ).to.equal( 'Insert image' );
+				expect( button.icon ).to.match( /<svg / );
+			} );
+
+			it( 'should have an arrow with the base properties set correctly', () => {
+				const buttonArrow = dropdown.buttonView.arrowView;
+
+				expect( buttonArrow.tooltip ).to.be.true;
+				expect( buttonArrow.label ).to.equal( 'Insert image' );
+			} );
```

Comment:
Obsolete per my comment in [codeblockui.js](https://github.com/ckeditor/ckeditor5/pull/11859/files?show-viewed-files=true&file-filters%5B%5D=#diff-a75a67319b38c5dc7c27b30eeaff970892cdd73e01b28920a569d161458ea8c7).